### DISCUSSION TITLE
9 Led option for 84/89

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#332] [Ideas]
We still don't have an option to turn on all the 9 led on the new frutto pucks on 84/89 mode, just 3 max.

**Comment by nomakewan** [comment]
Correct. I actually implemented a way to do this in the code already but it's commented out per the request of the other devs:

https://github.com/gpstar81/GPStar-proton-pack/blob/main/source/ProtonPack/ProtonPack.ino#L3064

In v6 since we're refactoring a lot of the menu options to be enums/structs instead of bools, the option for 84/89 cyclotron will be 1/3/All. But that won't be until the v6 menu refactor. For now, the code exists, it's ready to go, but there's no way to enable it.

**Comment by Toy203** [comment]
Actually, after trying 1 led with new frutto 9leds I'm completely satisfied. 3 leds already shapes like a small curve and it should not happening, 9 will be worse I believe. You'd need an heck of a diffuser to diffuse tjat amount of light and disguise the curve they make.
